src/Dockerfile.fedora

```diff
@@ -44,12 +48,18 @@ COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 # This image must forever use UID 26 for postgres user so our volumes are
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
+{% if spec.version == "16" %}
```

Comment:
This way of distinguishing between modularized and de-modularized versions is not really extensible. We will have  postgresql 17 in half a year, then new version every year and having so many conditions will be error-prone.

In demodularized fedoras we could install postgresql{% spec.version %}, etc.

Reply:
I would keep this format, at least for PG16. The new concept of packaging postgresql is entirely new, and it needs to be tested, so there could be slight changes in the feature.